ui/tests/e2e/connections.spec.js

```diff
@@ -151,6 +151,43 @@ test(
   },
 );
 
+test('Delete Kubernetes cluster connections', async ({ page }) => {
+  // Navigate to 'Connections' tab
+  await page.getByRole('tab', { name: 'Connections' }).click();
+  // Find the row with the connection to be deleted
+  const row = page
+    .locator('tr')
+    .filter({ hasText: 'kubernetes' })
+    .filter({ hasText: 'connected' })
+    .first();
```

Comment:
In Ci we have clusters name: `kind-cluster`, it will got discovered in the top of the tables. Because there are a full lifecyle from registering the cluster, please put this in the last of test cases in connections specs

https://github.com/meshery/meshery/blob/master/.github/workflows/build-ui-and-server.yml#L76

Reply:
Ok just to clarify, the `kind-cluster` will be the in the row of the text im looking for? So change above to: 

```
const row = page
    .locator('tr')
    .filter({ hasText: 'kind-cluster' })
    .filter({ hasText: 'connected' })
    .first();
```

is that correct!